qemu/tests/cfg/boot_from_virtiofs.cfg

```diff
@@ -0,0 +1,29 @@
+- boot_from_virtiofs:
+    no Host_RHEL.m7, Host_RHEL.m8, Host_RHEL.m9.u0, Host_RHEL.m9.u1
+    virt_test_type = qemu
+    type = boot_from_virtiofs
+    start_vm = no
+    file_system_package = "system-release vim-minimal systemd passwd dnf rootfiles sudo kernel  kernel-modules"
+    install_file_system_cmd = "dnf --installroot=%s --releasever=9 -y install ${file_system_package}"
+    create_initramfs_cmd = "cd %s ; dracut initramfs-virtiofs.img --early-microcode --add virtiofs --filesystems virtiofs"
+    check_selinux_cmd = "getenforce"
+    fs_passwd = virtiofs
+    set_passwd_cmd = "echo ${fs_passwd} | passwd --stdin root ; exit"
+    images = ""
+    cdroms = ""
+    nics = ""
+    filesystems = fs
+    fs_target = 'myfs'
+    fs_source_dir = virtiofs_root/
+    fs_driver = virtio-fs
+    fs_driver_props = {"bootindex": 1}
+    mem = 4096
+    guest_numa_nodes = "node0"
+    mem_devs = mem0
+    use_mem_mem0 = "no"
+    size_mem_mem0 = 4096M
+    numa_memdev_node0 = mem-mem0
+    mem-path_mem0 = /dev/shm
+    backend_mem_mem0 = memory-backend-file
+    share_mem = yes
+    kernel_params = "root=virtiofs:${fs_target} console=ttyS0"
```

Comment:
So should we define `shell_prompt` here?

Reply:
Yes, updated the code accordingly.Thanks a lot!